pkg/cvo/cvo.go

```diff
@@ -144,7 +144,7 @@ func New(
 
 		minimumUpdateCheckInterval: minimumInterval,
 		payloadDir:                 overridePayloadDir,
-		defaultUpstreamServer:      "http://localhost:8080/graph",
+		defaultUpstreamServer:      "https://api.openshift.com/api/upgrade_info/v1/graph",
```

Comment:
FWIW: Juan anticipated *upgrade_mgmt* as the prefix on the E-Mail thread on this topic. The previous anticipation was *update_mgmt*.